README.md

```diff
@@ -23,7 +23,7 @@ import validator from 'validator';
 Or, import only a subset of the library:
 
 ```javascript
-import isEmail from 'validator/lib/isEmail';
+import { isEmail, isFloat } from 'validator';
```

Comment:
These are not equivalent. The line you removed imports a single file/validator only. The line you've replaced it with imports the entire library and then selects two of the validators.